src/toolbar/toolbar-controller.js

```diff
@@ -59,7 +59,6 @@
 
         normalSizeClick() {
             this.window.restore();
-            this.window.resizeTo(defaultWidth, defaultHeight, 'top-right');
```

Comment:
I'm fairly sure this was in for a good reason. I think that there's a way to end up with the `restore()` method only returning to the compact size without this line, if I remember rightly. I'll test and see if that's still an issue.